YIF.Core.Domain/Repositories/LectorRepository.cs

```diff
@@ -70,6 +71,12 @@ public async Task<LectorDTO> GetLectorByUserAndIoEIds(string userId, string ioEI
             return _mapper.Map<LectorDTO>(lector);
         }
 
+        public async Task<LectorDTO> GetLectorByLectorIdAndIoEId(string lectorId, string ioEId)
```

Comment:
such method already exist in lines 68 - 72

Reply:
Fixed